phpstan.neon.dist

```diff
@@ -2,9 +2,9 @@ parameters:
     level: 1
     paths:
         - .
-    excludes_analyse:
+    excludePaths:
         - vendor/
+        - var/
```

Comment:
This path should never be committed into the repo.
However, whilst running phpunit, /var appears.

Reply:
I guess the `var/` directory appears in the project root because you are removing [`TestKernel::getBaseDir()`](https://github.com/hackzilla-project/TicketBundle/pull/223/files#diff-1405b52b9a93d6b25877321f62030ef1f01cd1a1563d03c55b71b8d8c2e55d61L182).

Reply:
yeah.  Though now I've had experience with supporting multiple versions of php/symfony, I'm tempted to revert that specific change.